generate_das/generate_das.py

```diff
@@ -94,54 +94,60 @@ def getYear():
     if actual_month == 1:
         actual_year = actual_year - 1
 
-pyautogui.sleep ( 0.3 )
+pyautogui.sleep(0.3)
 positionYear = {
-    2022: pyautogui.locateCenterOnScreen ( 'images/2022.png', confidence=0.95 ),
-    2023: pyautogui.locateCenterOnScreen ( 'images/2023.png', confidence=0.95 ),
-    2024: pyautogui.locateCenterOnScreen ( 'images/2024.png', confidence=0.95 ),
-    2025: pyautogui.locateCenterOnScreen ( 'images/2025.png', confidence=0.95 ),
-    2026: pyautogui.locateCenterOnScreen ( 'images/2026.png', confidence=0.95 ),
+    2022: pyautogui.locateCenterOnScreen('images/2022.png', confidence=0.95),
+    2023: pyautogui.locateCenterOnScreen('images/2023.png', confidence=0.95),
+    2024: pyautogui.locateCenterOnScreen('images/2024.png', confidence=0.95),
+    2025: pyautogui.locateCenterOnScreen('images/2025.png', confidence=0.95),
+    2026: pyautogui.locateCenterOnScreen('images/2026.png', confidence=0.95),
 }
 
-pyautogui.click ( positionYear[actual_year].x, positionYear[actual_year].y )
-pyautogui.sleep ( 1 )
+pyautogui.click(positionYear[actual_year].x, positionYear[actual_year].y)
+pyautogui.sleep(1)
 
 # CLICK SUBMIT
-pyautogui.press ( 'tab' )
-pyautogui.press ( 'enter' )
+pyautogui.press('tab')
+pyautogui.press('enter')
 
 # CLICK CHECKBOX MONTH
-loading_screen ( 'component_header_month' )
-positionMonth = {
-    1: pyautogui.locateOnScreen ( 'images/jan.png', confidence=0.95 ),
-    2: pyautogui.locateOnScreen ( 'images/fev.png', confidence=0.95 ),
-    3: pyautogui.locateOnScreen ( 'images/mar.png', confidence=0.95 ),
-    4: pyautogui.locateOnScreen ( 'images/abr.png', confidence=0.95 ),
-    5: pyautogui.locateOnScreen ( 'images/mai.png', confidence=0.95 ),
-    6: pyautogui.locateOnScreen ( 'images/jun.png', confidence=0.95 ),
-    7: pyautogui.locateOnScreen ( 'images/jul.png', confidence=0.95 ),
-    8: pyautogui.locateOnScreen ( 'images/ago.png', confidence=0.95 ),
-    9: pyautogui.locateOnScreen ( 'images/set.png', confidence=0.95 ),
-    10: pyautogui.locateOnScreen ( 'images/out.png', confidence=0.95 ),
-    11: pyautogui.locateOnScreen ( 'images/nov.png', confidence=0.95 ),
-    12: pyautogui.locateOnScreen ( 'images/dez.png', confidence=0.95 ),
+loading_screen('component_header_month')
+tab_presses = {
+    1: 9,
+    2: 11,
+    3: 13,
+    4: 15,
+    5: 17,
+    6: 19,
+    7: 21,
+    8: 23,
+    9: 25,
+    10: 27,
+    11: 29,
+    12: 9
 }
 
-month_pay = actual_month
+# Obtendo o mês atual
+current_month = datetime.now().month
 
-if not auto & month:
-    month_pay = month
-else:
-    month_pay = actual_month == 1 if 12 else actual_month - 1
+# Calculando o número de pressionamentos de tecla TAB necessários
+num_tab_presses = tab_presses[current_month - 1]
+
+# Pressionando a tecla TAB o número de vezes necessário
+for i in range(num_tab_presses):
+    pyautogui.press('tab')
```

Comment:
pyautogui.press ( 'tab', presses=6 )

Alterar o "for"  para a proprieda "presses"